common/_overrides.scss

```diff
@@ -7,20 +7,44 @@
     background-color: $hover-bg !important;
     color: $hover-color !important;
     border-color: $hover-border-color !important;
+    & span {
+      color: $hover-color !important;
+    }
+  }
+
+  & span {
+    color: $color !important;
   }
 }
 
 // Set the default color of links
-a:not(.btn),
+a:not(.btn, .help-link),
 a:not(.btn):hover,
 a:not(.btn):focus,
 a:visited:not(.btn):hover,
-a:visited:not(.btn):focus,
-a:not(.help-link) {
+a:visited:not(.btn):focus {
   color: $link-color;
 }
 
-.btn-primary {
+// Set color for course nav bar in Learning MFE
+.course-tabs-navigation a.nav-link{
+  @if variable-exists(course-nav-menu-color){
+    &:focus, &:hover, &:visited:hover, &:visited:focus, &.active {
+      color: $course-nav-menu-color;
+    }
+  }
+  @if variable-exists(course-nav-menu-border-bottom-color) {
+    &.active {
+      border-bottom-color: $course-nav-menu-border-bottom-color;
+    }
+  }
+}
+
+.btn-primary, .btn-brand {
   // Set the styles for the primary button
   @include btn-styles($btn-primary-bg, $btn-primary-color, $btn-primary-border-color, $btn-primary-hover-bg, $btn-primary-hover-color, $btn-primary-hover-border-color);
 }
+
+.btn-outline-primary {
```

Comment:
What about `btn-outline-secondary`? The "Next" button is styled, but the "Previous" one is not.
![image](https://user-images.githubusercontent.com/3189670/133936793-9151164a-4d87-4293-8571-cbf48dc28127.png)

Reply:
I have added a change just for the MFE in paragon overrides to use the same styling as btn-outline-primary(btn-secondary styles), as in classic LMS previous and next button use same styling